crates/zksync/core/src/vm/runner.rs

```diff
@@ -57,7 +58,7 @@ use super::{storage_view::StorageView, tracer::CheatcodeTracerContext};
 /// Maximum gas price allowed for L1.
 const MAX_L1_GAS_PRICE: u64 = 1000;
 
-type ZKVMResult<E> = EVMResultGeneric<rExecutionResult, E>;
+type ZKVMResult<E> = EVMResultGeneric<(Vec<rLog>, rExecutionResult), E>;
```

Comment:
Is this consistent with rEVM behavior that the logs are available in case of a revert?

Also might be cleaner returning a well typed struct
```rs
ZKExecutionResult { 
  execution_result: rExecutionResult, 
  logs: Vec<rLog>,
}
```

Reply:
Not 1000% sure, but when we have a function that reverts in EVM we are still able to see the logs, I think because we record them as they happen meanwhile here they are all at the end of the execution